modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/java/com/liferay/content/dashboard/document/library/internal/item/FileEntryContentDashboardItem.java

```diff
@@ -432,7 +439,25 @@ public String getUserName() {
 
 	@Override
 	public String getViewVersionsURL(HttpServletRequest httpServletRequest) {
-		return null;
+		return PortletURLBuilder.create(
```

Comment:
@Mikellorza, would it be possible to use _requestBackedPortletURLFactory.createRenderURL here?

Reply:
I deleted this methods in lasts commits.